docker/Dockerfile

```diff
@@ -252,8 +275,68 @@ RUN set -e; \
     rm -rf ~/.cache/pip; \
     truncate -s0 ~/.bash_history;
 
+# TODO(chenhao94) Build packages for arm64.
+#
+# For now we only build packages for amd64 because CI is running on amd64 machines.
+# Otherwise the docker image cross-compiling will be too slow.
```

Comment:
It's not "cross-compiling"

Reply:
"cross-platform-building"? I don't know what is the correct terminology here.

Reply:
You're blaming the wrong thing, it's not the docker image build that's slow, it's just your VM is slow.
"Otherwise the CI docker build on ARM emulation will be too slow."